Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -32,6 +32,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Data\" />
+    <Folder Include="Pages\" />

```

Comment:
what is this needed for?

Reply:
@srkirkland Sigh, these keep getting added when I use VS to add a view. Resharper adds it in the correct place